plugins/amazon.py

```diff
@@ -0,0 +1,38 @@
+import bottlenose
+import bitly_api
+from lxml import etree
+from util import hook, http
+
+@hook.api_key('amazon')
+@hook.command('amazon')
+@hook.command('a')
+def amazon(inp, api_key=None):
+    if api_key is None or 'AWS_KEY' not in api_key or 'SECRET_KEY' not in api_key or 'ASSOCIATE_TAG' not in api_key:
```

Comment:
Instead of `AWS_KEY` this should probably be `ACCESS_KEY` to clarify what it is.

Also, all the other configuration variables are lowercase.  These should probably be as well to be uniform.